src/niscope/templates/_grpc_stub_interpreter.py/fetch_array_measurement.py.mako

```diff
@@ -1,7 +1,2 @@
-
-    def fetch_array_measurement(self, channel_list, timeout, array_meas_function, measurement_waveform_size):  # noqa: N802
-        response = self._invoke(
-            self._client.FetchArrayMeasurement,
-            grpc_types.FetchArrayMeasurementRequest(vi=self._vi, channel_list=channel_list, timeout=timeout, array_meas_function_raw=array_meas_function.value),
-        )
-        return response.meas_wfm, [waveform_info.WaveformInfo(x) for x in response.wfm_info]
+<%page args="f, config, method_template"/>\
```

Comment:
I could add an override to use the default template and delete this and the LibraryInterpreter template. Would that be desirable or would it be better to keep this around?

Reply:
We can get rid of these templates? If so, that sounds great.